kerastuner/engine/multi_execution_tuner.py

```diff
@@ -72,13 +72,18 @@ def on_epoch_end(self, trial, model, epoch, logs=None):
         pass
 
     def run_trial(self, trial, *fit_args, **fit_kwargs):
+        save_freq = 'epoch'
+        if 'save_freq' in fit_kwargs:
+            save_freq = int(fit_kwargs.pop('save_freq'))
+
         model_checkpoint = keras.callbacks.ModelCheckpoint(
```

Comment:
Hm maybe instead, we could check to see if the user passed their own `ModelCheckpoint` object? If they did, instead of creating our own, we could patch the save directory on that object so that it saves to a different directory each time, and use that?

We do something similar for TensorBoard here: https://github.com/keras-team/keras-tuner/blob/master/kerastuner/engine/tuner.py#L272

Reply:
I see. Asking user to pass `ModelCheckpoint` object will be clearer and more consistent than allowing a specific keyword. Is that what you mean?